extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -355,7 +355,7 @@ function arraysearch( &$parser, $arrayid, $needle, $index=0, $yes=null, $no=null
 *    {{#arrayreset:}}
 *    {{#arrayreset:arrayid1,arrayid2,...arrayidn}}
 */
-   function arrayreset( &$parser, $arrayids) {
+   function arrayreset( $parser, $arrayids) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)